src/exports.cpp

```diff
@@ -20,10 +20,26 @@ BEGIN_RCPP
 END_RCPP
 }
 
+NumericMatrix get_empirical(List mmrm_fit, NumericVector theta, NumericVector beta, NumericMatrix beta_vcov, bool jackknife);
+RcppExport SEXP _mmrm_get_empirical(SEXP mmrm_fitSEXP, SEXP thetaSEXP, SEXP betaSEXP, SEXP beta_vcovSEXP, SEXP jackknifeSEXP) {
```

Comment:
```suggestion
RcppExport SEXP _mmrm_get_empirical(SEXP mmrm_fit_SEXP, SEXP thetaSEXP, SEXP betaSEXP, SEXP beta_vcovSEXP, SEXP jackknifeSEXP) {
```
etc to better read it?

Reply:
ok will do